roles/openshift_gce_cluster_inventory/tasks/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I'm going to change this to suck everything out of the current project that matches the configured tags into groups (so it'll be closer to the group definition).